CONTRIBUTING.md

```diff
@@ -126,12 +132,28 @@ are always welcome.
 
 ## Formatting the code
 
-We use `stylish-haskell` for Haskell code formatting. There is a CI
-script (TODO: link to it) that checks that the code is properly
-formatted.
+We use `stylish-haskell` >= 0.11.0.0 for Haskell code formatting. There is a CI
```

Comment:
I think we should update this to `>= 0.14.3.0`, which is the version that CI uses (IIRC)

Reply:
Thanks, done!